sklearn/base.py

```diff
@@ -621,8 +622,11 @@ def get_shape(self, i):
 
         Returns
         -------
-        shape : tuple (int, int)
-            Number of rows and columns (resp.) in the bicluster.
+        n_rows : int
+            Number of rows in the bicluster.
+
+        n_cols : int
+            Number of columns in the bicluster.
```

Comment:
I'm not sure if this is less confusing than the `shape: tuple` for the users. WDYT @NicolasHug

Reply:
I think in general we separate the entries like that (numpy does it too)
For 2-uples it's OK to merge them to IMHO

Reply:
I decided to separate in two entries because is the approach used in the `get_indices` method. Nevertheless, LGTM any of these approaches.

Which approach do you prefer?